build_daemon/lib/data/server_log.dart

```diff
@@ -7,6 +7,7 @@ import 'package:built_value/serializer.dart';
 
 part 'server_log.g.dart';
 
+/// A log 
```

Comment:
remove this one since it's not adding new information.

Reply:
Oops. Meant to fill it in more but ultimately decided it really didn't need a comment.